frontend/src/services/apiService.ts

```diff
@@ -69,6 +69,18 @@ export const useApiFetch = createFetch({
         })
       }
 
+      // this error is sometimes throws when previous requests are automatically aborted as
+      // some of the data changed and refetch is true. It seems to only be throws when the request
+      // is aborted before it was actually send.
+      // We ignore this error as it (for some odd reason) isn't replaced once the second request finishes
+      // successfully
+      if (fetchContext.error.name === "AbortError") {
```

Comment:
Question: I created unit tests for `useApiFetch`, is this a behavior we can/should test there?

Reply:
hm... I'm not sure if i'm able to mock the specific behaviour that is causing this, but i could give it a try, and at least i can add a test for `AbortError`s in general.